src/lib/system-info.ts

```diff
@@ -120,28 +120,22 @@ const significantChange: { [key in keyof SystemInfo]?: number } = {
 	memory_usage: 10,
 };
 
-export function filterNonSignificantChanges(
-	past: Partial<SystemInfo>,
-	current: SystemInfo,
-): Array<keyof SystemInfo> {
-	return Object.keys(
-		_.omitBy(current, (value, key: keyof SystemInfo) => {
-			// If we didn't have a value for this in the past, include it
-			if (past[key] == null) {
-				return true;
-			}
-			const bucketSize = significantChange[key];
-			// If we don't have any requirements on this value, include it
-			if (bucketSize == null) {
-				return true;
-			}
-
-			return (
-				Math.floor((value as number) / bucketSize) !==
-				Math.floor((past[key] as number) / bucketSize)
-			);
-		}),
-	) as Array<keyof SystemInfo>;
+export function isSignificantChange(
```

Comment:
I changed to a single function that just returns whether the change is significant or not, it's more versatile and imo it's also simpler due to avoiding the weird keys/omitBy combo, plus the previous function name was a bit misleading as it did not filter but instead returned a list of keys